poll.js

```diff
@@ -0,0 +1,150 @@
+'use strict'
+var imgs = ['bag.jpg',
+            'banana.jpg',
+            'bathroom.jpg',
+            'boots.jpg',
+            'breakfast.jpg',
+            'bubblegum.jpg',
+            'chair.jpg',
+            'cthulhu.jpg',
+            'dog-duck.jpg',
+            'dragon.jpg',
+            'pen.jpg',
+            'pet-sweep.jpg',
+            'scissors.jpg',
+            'shark.jpg',
+            'sweep.png',
+            'tauntaun.jpg',
+            'unicorn.jpg',
+            'usb.gif',
+            'water-can.jpg',
+            'wine-glass.jpg'];
+
+var choices = [];
+choices.push(document.getElementById('picture0'));
+choices.push(document.getElementById('picture1'));
+choices.push(document.getElementById('picture2'));
+var results = document.getElementById('results');
+var rlist = document.getElementById('resultslist');
+
+/**
+ * [Image description]
+ * @param {[type]} file [description]
+ */
+function Image(file) {
+  this.score = 0; //times selected
+  this.appeared = 0; //times appeared on the screen
+  this.file = file; //image path
+}
+
+Image.prototype.getPath = function() {
+  return 'img/' + this.file;
+}
+
+Image.prototype.tally = function(chosen) {
+  this.appeared++;
+  if(chosen){this.score++;}
+}
+
+Image.prototype.getPercent = function() {
+  if(this.score === 0) return 0;
+  var prc = (this.score / this.appeared) * 100;
+  return Math.round(prc);
+}
+
+/**
+ * [Tracker description]
+ * @type {Object}
+ */
+var Tracker = {
+  round: 0,
+  maxRounds: 25, //15?
+  images: [],
+
+  addImage: function(image) {
+    this.images.push(image);
+  },
+
+  //swaps two Images in the
+  swapImages: function(im1, im2) {
+    if(im1 === im2) {return;}
+    if(im1 >= this.images.length || im2 >= this.images.length){return;}
+
+    var temp = this.images[im1];
+    this.images[im1] = this.images[im2];
+    this.images[im2] = temp;
+  },
+
+  //shuffles random images into the first three image slots
+  shuffle: function() {
```

Comment:
Do you think that you need this? Can you think of a way to solve the problem that doesn't involve shuffling all your base data?